lib/cmdUtil.js

```diff
@@ -0,0 +1,83 @@
+
+var inquirer = require('inquirer');
+var fs = require('fs');
+
+var stravaConfig = './data/strava_config';
+var stravaConfigTemplate = './strava_config';
+var stravaApiUrl = 'https://www.strava.com/settings/api#_=_';
+
+var strava = require('../index.js');
+
+/**
+ * Generates the token to access the strava application
+ */
+module.exports.generateNewToken =  function () {
+
+    console.log('Before processing, you shall fill your strava config with client id and secret provided by Strava:\n' + stravaApiUrl);
+
+
+    inquirer
+        .prompt(
+            [
+                {
+                    type: 'input',
+                    name: 'clientId',
+                    message: 'What is your strava client id?'
+                },
+                {
+                    type: 'input',
+                    name: 'clientSecret',
+                    message: 'What is your strava client secret?'
+                }
+            ])
+        .then(function (answers) {
+
+            // We copy the strava config file
+            try {
+                fs.mkdirSync('data');
+            } catch (e) {
```

Comment:
We shouldn't swallow the exception here. If the call fails because the directory already exists, that's fine. However, if it fails e.g. due to missing change rights, we need to report this and cancel our processing. In this case, writing the config file will likely also fail.

This could look something like this:
```
try {
  fs.mkdirSync('data');
} catch (e) {
  if (e.code !== 'EEXIST') throw e;
}
```